src/v2/ion/IonResponseHelper.js

```diff
@@ -151,7 +152,16 @@ const isIonErrorResponse = (response = {}) => {
   return response.version;
 };
 
+const isTerminalError = (res = {}) => {
```

Comment:
Note: This looks like something we should move to idx-js in the future.

Reply:
This is evaluating the ION response to determine if it encodes a terminal error view.  It would be cool to be able to evaluate the IDX response and know the same thing. I could not find a way that did not look ugly, so I went this path, using the "transformed" ION response.

Reply:
No, this looks correct, I just think that idx-js should hold this logic and expose if a response is terminal (my original idea for doing so was shot down, but now we have evidence that the logic is needed).  

So in the future idx-js can hold this logic and the widget can just check to see if a response is flagged as terminal.

Reply:
...I also think all the ION transformation should ALSO be done in idx-js.